man/tpm2_policypassword.1.md

```diff
@@ -0,0 +1,79 @@
+% tpm2_policypassword(1) tpm2-tools | General Commands Manual
+%
+% AUGUST 2018
+
+# NAME
+
+**tpm2_policypassword**(1) - Enables binding a policy to the authorization value
+ of the authorized TPM object.
+
+# SYNOPSIS
+
+**tpm2_policypassword** [*OPTIONS*]
+
+# DESCRIPTION
+
+**tpm2_policypassword**(1) Enables a policy that requires the object's 
+authentication passphrase be provided. This is equivalent to authenticating 
+using the object passphrase in plaintext, only this, enforces it as a policy. 
+It is a necessary way of requiring object's auth passphrase knowledge when the 
+object attributes mandate authorization only with policies; i.e. objects with 
+attribute "userwithauth = 0".
+
+Requires extended session support using a resource manager.
```

Comment:
This isn't 100% true, it also works if you **don't** use an RM, so the statement should be: If using a resource manager (RM), then one supporting extended sessions, like `[tpm2-abrmd](https://github.com/tpm2-software/tpm2-abrmd)` is required.